### PR TITLE
Adding ability for op to see the inventory of chests with a block above it

### PR DESCRIPTION
**Pull request by Maxopoly**
Adressing https://github.com/Civcraft/Humbug/issues/58

@rourke750

### CODE REVIEW

File: src/main/java/com/untamedears/humbug/Humbug.java [hunk 1]
```diff
@@ -2511,6 +2513,35 @@ public void onSignLoads(ChunkLoadEvent event) {
       }
     }
   }
+  
+  @EventHandler(priority = EventPriority.HIGHEST)
+  public void adminAccessBlockedChest(PlayerInteractEvent e) {
+	  if (!e.getPlayer().isOp()) {
```
Comment (rourke750) [inline src/main/java/com/untamedears/humbug/Humbug.java]: also add a permission check here if you could as well.  Otherwise looks perfect.

**Comment by rourke750** [comment]
Also can you increment the version in the pom.

**Comment by rourke750** [comment]
ok to test

**Comment by rourke750** [comment]
@Maxopoly Did you test this?

**Comment by Maxopoly** [comment]
I did, it didn't work, I fixed it here: https://github.com/Civcraft/Humbug/commit/7b059c7561b72ca4af7022ede9974f2d3aa343bb

I already tested it with that and it's working, I also changed it so you dont get a copy of the chest inventory, but instead get the chest inventory itself so you can actually take items out and put items in there.

**Pull request merged**